references/threat-protect/test/integration/features/step_definitions/init.js

```diff
@@ -0,0 +1,37 @@
+/**
+  Copyright 2021 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+const apickli = require('apickli')
+const {
+  Before,
+  setDefaultTimeout
+} = require('cucumber')
+
+const basePath = '/threats/v1'
+const depth = 5
+const length = 5
+const height = 5
+const width = 5
+
+Before(function() {
```

Comment:
can you add this to your file?

```js
/* eslint-disable new-cap */
```

see https://github.com/apigee/devrel/blob/25726ea0b80e2f46ed3fd4a004d30c1a62e6e654/references/cicd-pipeline/test/integration/support/init.js#L18

Reply:
Done